File: fizz-buzz/fizz-buzz.go
// String conversion is a whole nother import.
package main
import "fmt"
func main(){for i:=1;i<101;i++{a:="";if(i%3==0){a="Fizz"};if(i%5==0){a+="Buzz"};if(a==""){fmt.Println(i)}else{fmt.Println(a)}}}

File: fizz-buzz/fizz-buzz.fs
// Trying to do this in a normal for loop was an error nightmare.
let a=seq{for i in 1..100 do if i%15=0 then yield"FizzBuzz"elif i%3=0 then yield"Fizz"elif i%5=0 then yield"Buzz"else yield i.ToString()}
for x in a do printfn"%s"x
